### PR TITLE
Add support for Clojure with clj-kondo

### LINKED ISSUE #111: Clojure support
Using https://github.com/borkdude/clj-kondo

### PR DESCRIPTION
**Pull request by goedelsoup**
Fixes #111 by adding the clj-kondo CLI to the runtime environment and binding the following extensions to the linter: clj, cljc, cljs, edn.

### CODE REVIEW

File: lib/linter.sh [hunk 1]
```diff
@@ -2358,6 +2396,20 @@ if [ "$VALIDATE_DOCKER" == "true" ]; then
...
+###################
+if [ "$VALIDATE_CLOJURE" == "true" ]; then
+  #################################
+  # Get Clojure standard rules #
+  #################################
+  GetStandardRules "clj-kondo"
+  #########################
+  # Lint the Clojure files #
+  #########################
+  LintCodebase "CLOJURE" "clj-kondo" "clj-kondo --config $CLOJURE_LINTER_RULES" ".*\.\(clj\)\$" "${FILE_ARRAY_CLOJURE[@]}"
```
[the remark this answers is not in the document]
Comment (goedelsoup) [reply lib/linter.sh:2410]: Yup! Good catch.

**Comment by admiralAwkbar** [comment]
@goedelsoup Can you add the docs for how to tune the linter?
Like for how to disable for inline, whole file, block?

you can see the list on https://github.com/github/super-linter/wiki/Adding-new-language-support

Ping me when you have it in, and I will pull this in for formal CI

**Comment by borkdude** [comment]
Clj-kondo config documentation is located here: 

https://github.com/borkdude/clj-kondo/blob/master/doc/config.md

Would it be good to add a link to that?

**Comment by goedelsoup** [comment]
@borkdude Just updated, noting that they would be more useful than what could be covered here.

@admiralAwkbar I believe this should be good to go.

**Comment by admiralAwkbar** [comment]
Pulling this fork into a branch for full ci

**Pull request closed without merging**